Docker/itk-base/Dockerfile

```diff
@@ -16,7 +17,11 @@ RUN curl -O https://cmake.org/files/v3.4/cmake-3.4.3.tar.gz && \
     cd /tmp/cmake-build && \
     ../cmake-3.4.3/bootstrap && \
     make -j$(nproc) && \
-    ./bin/cmake -DCMAKE_BUILD_TYPE:STRING=Release . && \
+    cd .. && \
```

Comment:
Ideally, the official CMake binaries should be built with manylinux so that we can run on most of linux distrib.

Otherwise, you could pip install cmake (or download and extract the wheel). It would speed up built time